census/src/test/scala/model/RssdToMsaSpec.scala

```diff
@@ -0,0 +1,13 @@
+package model
+
+import hmda.census.model.RssdToMsa
+import org.scalatest.{ MustMatchers, WordSpec }
+
+class RssdToMsaSpec extends WordSpec with MustMatchers {
+  "RssdToMsa" must {
+    "map values correctly" in {
```

Comment:
I thought about doing this, but wasn't comfortable with tests relying on the contents of a file that's going to updated and changed.